tests/test_makeRawVisitInfoViaObsInfo.py

```diff
@@ -0,0 +1,85 @@
+# This file is part of obs_base.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+import unittest
+
+from astropy.time import Time
+
+from lsst.obs.metadata import FitsTranslator
+from lsst.daf.base import DateTime
+
+from lsst.obs.base import MakeRawVisitInfoViaObsInfo
+
+
+class NewTranslator(FitsTranslator):
+    _trivialMap = {
+        "exposure_time": "EXPTIME",
+        "exposure": "EXP-ID",
+    }
+
+    def to_location(self):
+        return None
+
+    def to_detector_exposure_id(self):
+        return self.to_exposure()
```

Comment:
Docstrings or comments on this helper class would be helpful (doesn't need anything huge or formal, just an explanation of the role this plays in the tests).